File: BOJ_단계/조건문/2525.cpp
#include<iostream>

using namespace std;

int main() {
    int A, B;
    int min;
    
    int plus_H;
    
    cin >> A >> B;
    cin >> min;

    B = B + min;
    plus_H = B / 60;
    B = B % 60;
    A = A + plus_H;
    if (A >= 24) {
        A = A - 24;
    }

    cout << A << " " << B;

}

File: BOJ_단계/조건문/2525.py
A, B = map(int, input().split())
C = int(input())


plus_H = int((B + C) / 60)
B = (B + C) % 60

A = A + plus_H

if (A >= 24):
    A = A - 24

print(str(A) + " " + str(B))